Makefile

```diff
@@ -165,7 +169,13 @@ vm: $(VM_IMAGE)
 # sure VM_IMAGE is as close as possible to the host!
 vm-local-repos: vm
 	bots/image-customize -v \
-		--upload /etc/yum.repos.d:/etc/yum.repos.d/ \
+		--run-command "rm -rf /etc/yum.repos.d" \
+		$(TEST_OS)
+	bots/image-customize -v \
```

Comment:
There's no need to call it twice. This means an extra shutdown and boot for no reason, you can specify arbitrarily many command line options to image-customize.

Reply:
Actually there is. When I group the options into 1 command first new repos are uploaded and then the `rm` is executed deleting the newly uploaded files. We want to make sure that before continuing with the tests the VM contains only the specified repositories and nothing else!

Reply:
OK -- you could upload it to somewhere else, and move it afterwards, but if you don't mind the extra waiting this works as well. We recently discussed that in PM about whether i-customize should always start fresh or use an existing overlay (the latter happens now)